.github/workflows/common-frontend.yml

```diff
@@ -58,6 +58,11 @@ jobs:
         with:
           path: "~/.cache/Cypress"
           key: ${{ runner.os }}-cypress-cache-${{ hashFiles('frontend/package.json') }}
+      - name: Install Cypress (cache was not found)
```

Comment:
Tää `(cache was not found)` näyttää ehkä vähän hassulta stepin nimessä. Oisko ihan vaan `Install Cypress`?
![Screenshot 2022-12-15 at 13 14 47](https://user-images.githubusercontent.com/18038608/207845801-eda44dcb-693c-431a-937a-84fbac9fd356.png)

Reply:
joo, katoin samaa ku ajoin noita. Typistetään tohon